pkgs/development/libraries/science/math/mongoose/default.nix

```diff
@@ -16,17 +19,27 @@ stdenv.mkDerivation rec {
     sha256 = "0ymwd4n8p8s0ndh1vcbmjcsm0x2cc2b7v3baww5y6as12873bcrh";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/ScottKolo/Mongoose/commit/39f4a0059ff7bad5bffa84369f31839214ac7877.patch";
```

Comment:
Adding a descriptive `name` would be helpful (e.g. `name = "disable-coverage-option"`)
Can it be removed with the next release (maybe add a comment here)?

Reply:
Done.